src/registry/registryadd.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react';
+import { Card, Container, Row, Col, Modal } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios';
+import PlumFilledButton from '../Common/styled/buttonstyles/PlumFilledButton';
+import PlumButton from '../Common/styled/buttonstyles/PlumButton';
+
+export default function ResgistryAdd() {
+  const [registry, setRegistry] = useState([]);
+  const [registryItem, setRegistryItem] = useState({});
+  const [modalOn, setModalOn] = useState(false);
+  const [claimed, setClaimed] = useState([]);
+
+  const handleClose = () => setModalOn(false);
+  // const handleShow = () => setModalOn(true);
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setRegistryItem((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  const handleClick = (url) => {
+    window.location.assign(url);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { category } = registryItem;
+    axios.post('/registry', registryItem).then((results) => {
+      const { meta } = results.data;
+
+      setRegistry((prevState) => [
+        ...prevState,
+        { url: meta.url, title: meta.title, category: category },
+      ]);
+    });
+  };
+
+  return (
+    <Container>
+      <Row className="mt-5">
+        <Col md={6}>
+          <h1 style={{ fontSize: '64px', fontFamily: 'Alex Brush' }}>
+            Registry
+          </h1>
+        </Col>
+        <Col md={3} className="mr-1 pr-1">
+          <div style={{ float: 'right' }}>
+            <h2
+              style={{
+                fontSize: '24px',
+                fontFamily: 'Faustina serif',
+              }}
+            >
+              {registry.length} {registry.length === 1 ? 'Gift' : 'Gifts'}{' '}
+              Requested
+            </h2>
+            {claimed.length} Purchased
+          </div>
+        </Col>
+        <Col md={3} className="ml-1">
+          {modalOn ? (
+            <PlumFilledButton type="button" className="float-right">
+              add gifts
+            </PlumFilledButton>
+          ) : (
+            <PlumButton
+              onClick={() => setModalOn(true)}
+              type="button"
+              className="float-right"
+            >
+              add gifts
+            </PlumButton>
+          )}
+        </Col>
+      </Row>
+      <Row>
+        <h2
+          style={{
+            fontSize: '24px',
```

Comment:
not big but for font styles make sure to use the styled text in /common/styled/textstyles/etc